docs/docs/upgrade/network-api-migration.md

```diff
@@ -64,6 +64,9 @@ The following table summarizes the API transformations when using the network AP
     <pre>
 {
     “container”: {
+      "docker": {
+        "image": "foo"
```

Comment:
please include the `image` field in the `docker` object in the first col as well